.editorconfig

```diff
@@ -11,117 +15,128 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[*.md]
-insert_final_newline = false
-trim_trailing_whitespace = false
```

Comment:
Please re-add these lines

Reply:
Should already be under [*], 

```
[*]
indent_style = space
indent_size = 4
end_of_line = lf
charset = utf-8
trim_trailing_whitespace = true
insert_final_newline = true
```

Reply:
indeed we want to negate those two settings from the default `[*]` block, because

- two trailing whitespaces equal a linebreak in rendered markdown
- i can't remember right now why we would not want a final newline in markdown files. I hope i added this for a reason :D